controllers/nginx/pkg/cmd/controller/metrics.go

```diff
@@ -40,7 +42,10 @@ func (em exeMatcher) MatchAndName(nacl common.NameAndCmdline) (bool, string) {
 }
 
 func (n *NGINXController) setupMonitor(args []string) {
-	pc, err := newProcessCollector(true, exeMatcher{"nginx", args})
+	var enableVts = true
```

Comment:
@aledbf when the nginxController starts the configmap are empty.  Could I change the enable-vts-metrics from annotation to an argument?